### PR TITLE
task2 done

### PR DESCRIPTION
**Pull request by serhatkaya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,10 @@
-# Shop Angular Cloudfront
+# Manual deployment
 
-Angular version: ~12.
+- https://d3p56wgg4n9n8s.cloudfront.net
```
Comment (accmeboot) [inline README.md:3]: Cloud from returns 403 status code, and response is in XML for some reasons

File: .serverless/cloudformation-template-create-stack.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
+{
```
Comment (accmeboot) [inline .serverless/cloudformation-template-create-stack.json:1]: Could you please elaborate on how these JSON files are used?

Comment (serhatkaya) [reply .serverless/cloudformation-template-create-stack.json:1]: This is an autogenerated JSON file. You can investigate serverless docs.

File: README.md (hunk 1 above)
Comment (serhatkaya) [reply README.md:3]: Fixed

**Pull request merged**